packages/swingset-runner/demo/zoeTests/swingset.json

```diff
@@ -1,5 +1,10 @@
 {
   "bootstrap": "bootstrap",
+  "bundles": {
+    "zcf": {
+      "sourcePath": "../../../zoe/src/contractFacet.js"
```

Comment:
Hm. I don't see any way to avoid this package-crossing pathname and still use a static JSON config file, but this won't work anywhere outside of the monorepo. Our packages are nominally separate, and for all the other ones, you could download an NPM tarball of e.g. swingset-N.N.N and unpack it and run all the tests/examples no matter where the tarball got unpacked. The other swingset-runner demos use zoe by an import (which needs a `devDependency` in package.json, by the way), but this one `../../..` assumes even more about the nearby filesystem.

For cosmic-swingset, I'd like to see Zoe export a function by which you can get the absolute pathname (which zoe computes with `require.resolve` or some eventual `import.meta.url` equivalent), and then the cosmic-swingset *non*-static config-making code can import that from zoe and ask zoe for the filename to use for the bundle. That ought to work regardless of where the Zoe package is located.

So don't change this here, but let's limit the cases where we use something that reaches up out of the package subdirectory, and which wouldn't work if the monorepo were broken up.

Reply:
This is getting a bit out of the timeline, but a later change coming soon to a PR near you, based on a suggestion by Kate, adds the ability to say `sourcePath: "@agoric/zoe/src/contractFacet"` and have it do the right thing.